src/runners/http_service_runner.ts

```diff
@@ -28,41 +28,56 @@ process.on('unhandledRejection', err => {
     }
 });
 
+if (require.main === module) {
+    (async () => {
+        const provider = providerUtils.createWeb3Provider(defaultConfig.ETHEREUM_RPC_URL);
+        const dependencies = await getDefaultAppDependenciesAsync(provider, defaultConfig);
+        await runHttpServiceAsync(dependencies, defaultConfig);
+    })().catch(error => logger.error(error));
+}
+
 /**
  * This service handles the HTTP requests. This involves fetching from the database
  * as well as adding orders to mesh.
+ * @param dependencies If no mesh client is supplied, the HTTP service will start without it.
+ *                     It will provide defaults for other params.
  */
-(async () => {
-    const connection = await getDBConnectionAsync();
-    const app = express();
+export async function runHttpServiceAsync(
+    dependencies: AppDependencies,
+    config: { HTTP_PORT: string; ETHEREUM_RPC_URL: string },
```

Comment:
is `ETHEREUM_RPC_URL` needed here if the provider is in the AppDependencies?

Reply:
ah good catch. removing!